easybuild/framework/easyconfig/parser.py

```diff
@@ -45,6 +45,59 @@
 # alternate easyconfig parameters, and their non-deprecated equivalents
 ALTERNATE_PARAMETERS = {
     # <new_param>: <equivalent_param>,
+    'build_deps': 'builddependencies',
+    'build_in_install_dir': 'buildininstalldir',
+    'build_opts': 'buildopts',
+    'build_stats': 'buildstats',
+    'clean_up_old_build': 'cleanupoldbuild',
+    'clean_up_old_install': 'cleanupoldinstall',
+    'configure_opts': 'configopts',
```

Comment:
Any reason we don't abbreviate `configure` to `config` (or even `cfg`, but that's a stretch)?

```suggestion
    'config_opts': 'configopts',
```

Reply:
we're using `config` for "configuration" in the EB configuration options, e.g.:
```
--avail-config-file-constants
--config-files
--ignore-config-files
--job-backend-config
```

Reply:
The context there is different, it's fine is we used `config` for both imho.

It's clear that `config_opts` in an easyconfig file specified options for the configure command (which could be `cmake`, it's definitely not always `configure`).

Reply:
As discussed, let's stick to `configure_opts` (to align with `configure` step)